neon/pupil-cloud/offset-correction/index.md

```diff
@@ -0,0 +1,19 @@
+# Offset Correction on Pupil Cloud
+
+For some subjects, you may find a constant offset in their gaze estimates. This gaze offset can be [compensated for in the Neon Companion app](https://docs.pupil-labs.com/neon/data-collection/offset-correction/) before starting a recording, or post hoc in Pupil Cloud as shown below. Click on the gaze offset icon to start the gaze offset correction.
+
+![Offset correction on Cloud header image](./offset-cloud-timeline.png)
+
+Once you enter the `Edit Gaze Offset` view,  simply drag the gaze circle to apply the correction. See the video below for reference.
```

Comment:
The video does not actually show how to drag the gaze circle to apply the correction, does it? Would it make sense to demonstrate this interaction the video?

Reply:
Yes - I'm adding it!